### ISSUE TITLE
Android Manifest merger conflict

### ISSUE THREAD

**Issue by MrOnyszko** [Karn/sentry#2]
This [line](https://github.com/Karn/sentry/blob/42a111cca38789cb6e0e703cd1a3683c16a78507/library/src/main/AndroidManifest.xml#L27) could be redundant and it causes an issue with Android manifest merger.

```

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':mobile-ui:processDebugManifest'.
> Manifest merger failed : Attribute application@theme value=(@style/AppTheme.NoActionBar) from AndroidManifest.xml:17:9-52
  	is also present at [io.karn:sentry:0.0.1] AndroidManifest.xml:34:18-74 value=(@style/Theme.AppCompat.Light.NoActionBar).
  	Suggestion: add 'tools:replace="android:theme"' to <application> element at AndroidManifest.xml:10:5-28:19 to override.

```

**Comment by Karn** [comment]
Hey @MrOnyszko,

Do you happen to have a small reproducible project that can I can use to re-create the problem?

**Comment by Karn** [comment]
You can use the following configuration to import the test branch in #3 

`implementation "io.karn:sentry:42a111cca3"` where `42a111cca3` is the commit hash associated with the latest commit in the above PR.

Just tag me once you've verified that it works!

Thanks,
Karn.

**Comment by Karn** [comment]
Now fixed in `0.0.2`.
